nuxt.config.ts

```diff
@@ -9,10 +9,19 @@ export default defineNuxtConfig({
     },
   },
   nitro: {
-    preset: 'vercel-edge'
+    preset: 'vercel-edge',
```

Comment:
vamos aproveitar e mudar esse preset aqui pra vercel normal tambem? vercel-edge usa as edge functions da vercel (que tem algumas particularidades)
```suggestion
    preset: 'vercel',
```